vm_core.h

```diff
@@ -452,6 +452,11 @@ struct rb_iseq_constant_body {
 #endif
     } insns_info;
 
+    struct {
+        uint8_t *insns;
+        unsigned int size;
```

Comment:
What's different between `original_insns.size` and `iseq_size`?

Reply:
`original_insns.size` is the count of instructions in this iseq. `iseq_size` is the count of instructions + the total count of operands.